src/schema.js

```diff
@@ -26,14 +27,14 @@ const Schema = buildSchema(`
   }
 
   type Query {
+    fromWebsite(websiteUrl: String!): [Feed]
```

Comment:
This will make a request like the one below be valid. I think we should create a separate type for the search result list.

```graphql
fromWebsite(websiteUrl: "...") {
  title
}
```